packages/voice/src/rest/v1/applications/applications-api.ts

```diff
@@ -0,0 +1,216 @@
+      url,
+      requestOptions,
+      apiName: this.apiName,
+      operationId: 'QueryNumber',
+    });
+  }
+
+  /**
+   * Get Callback URLs
+   * Returns any callback URLs configured for the specified application.
+   * @param { GetCallbackURLsRequestData } data - The data to provide to the API call.
+   */
+  public async getCallbackURLs(data: GetCallbackURLsRequestData): Promise<GetCallbacks> {
+    this.client = this.getSinchClient();
+    const getParams = this.client.extractQueryParams<GetCallbackURLsRequestData>(data, [] as never[]);
+    const headers: { [key: string]: string | undefined } = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    };
+
+    const body: RequestBody = '';
+    const basePathUrl = `${this.client.apiClientOptions.basePath}/v1/configuration/callbacks/applications/${data['applicationkey']}`;
+
+    const requestOptions = await this.client.prepareOptions(basePathUrl, 'GET', getParams, headers, body);
+    const url = this.client.prepareUrl(requestOptions.basePath, requestOptions.queryParams);
+
+    return this.client.processCall<GetCallbacks>({
+      url,
+      requestOptions,
+      apiName: this.apiName,
+      operationId: 'GetCallbackURLs',
+    });
+  }
+
+  /**
+   * Get Numbers
+   * Get information about your numbers. It returns a list of numbers that you own, as well as their capability (voice or SMS). For the ones that are assigned to an app, it returns the application key of the app.
+   * @param { GetNumbersRequestData } data - The data to provide to the API call.
+   */
+  public async getNumbers(data: GetNumbersRequestData): Promise<GetNumbersResponseObj> {
+    this.client = this.getSinchClient();
+    const getParams = this.client.extractQueryParams<GetNumbersRequestData>(data, [] as never[]);
+    const headers: { [key: string]: string | undefined } = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    };
+
+    const body: RequestBody = '';
+    const basePathUrl = `${this.client.apiClientOptions.basePath}/v1/configuration/numbers`;
+
+    const requestOptions = await this.client.prepareOptions(basePathUrl, 'GET', getParams, headers, body);
+    const url = this.client.prepareUrl(requestOptions.basePath, requestOptions.queryParams);
+
+    return this.client.processCall<GetNumbersResponseObj>({
+      url,
+      requestOptions,
+      apiName: this.apiName,
+      operationId: 'GetNumbers',
+    });
+  }
+
+  /**
+   * Un-assign number
+   * Un-assign a number from an application.
+   * @param { UnassignNumberRequestData } data - The data to provide to the API call.
+   */
+  public async unassignNumber(data: UnassignNumberRequestData): Promise<void> {
+    this.client = this.getSinchClient();
+    const getParams = this.client.extractQueryParams<UnassignNumberRequestData>(data, [] as never[]);
+    const headers: { [key: string]: string | undefined } = {
+      'Content-Type': 'application/json',
+      'Accept': '',
```

Comment:
Is it useful to set an empty value ?

Reply:
Indeed, I have a bug in the generator, the header should be filled

Reply:
Fixed in DEVEXP-252: https://github.com/sinch/sinch-sdk-node/pull/7/files/db1c8265d1e4bf2e6b377d0e9f7c7941c734aa75#diff-c3975f130147ae5849c97cb47528a6987e58310afcd92329286b6ff93c8650d9L142